File: PHP/Events/GetEventPreliminaryBookingInfoResult.php
<?php
namespace LTD_API;

class GetEventPreliminaryBookingInfoResult
{
    //array of PreliminaryPerformanceInfo
    private $performances;
    
    public function getPerformances()
    {
        return $this->performances;
    }
    public function setPerformances($performances)
    {
        $this->performances = $performances;
    }
}

File: Java/Events/GetEventPreliminaryBookingInfoResult.java
package LTD_API;

import java.util.List;

public class GetEventPreliminaryBookingInfoResult
{
    private List<PreliminaryPerformanceInfo> performances;
    
    public List<PreliminaryPerformanceInfo> getPerformances()
    {
        return performances;
    }
    public void setPerformances(List<PreliminaryPerformanceInfo> performances)
    {
        this.performances = performances;
    }
}

